vm/devices/net/net_consomme/consomme/src/icmp.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#![allow(unsafe_code)]
```

Comment:
Is it possible to move the allow(unsafe_code) to right before the unsafe block, and to add a basic `// UNSAFTEY:` comment so we can remove the clippy allow as well?

Reply:
Lets not bypass the lints and automation that require us to document our unsafe code 😅

Please ensure you are documenting these new unsafe blocks, prior to landing this PR